packages/venia-ui/lib/components/LegacyMiniCart/product.js

```diff
@@ -14,6 +14,8 @@ import ProductOptions from './productOptions';
 import Section from './section';
 import { gql } from '@apollo/client';
 
+const QUANTITY_OPERATOR = 'x';
```

Comment:
Looks like this may be a special character `×` instead of the letter `x`

Reply:
Oops! You're good :D